sklearn/tree/tests/test_tree.py

```diff
@@ -260,22 +260,22 @@ def test_iris():
             "".format(name, criterion, score))
 
 
-def test_boston():
-    # Check consistency on dataset boston house prices.
+def test_diabetes():
+    # Check consistency on diabetes dataset.
 
     for (name, Tree), criterion in product(REG_TREES.items(), REG_CRITERIONS):
         reg = Tree(criterion=criterion, random_state=0)
-        reg.fit(boston.data, boston.target)
-        score = mean_squared_error(boston.target, reg.predict(boston.data))
+        reg.fit(diabetes.data, diabetes.target)
+        score = mean_squared_error(diabetes.target, reg.predict(diabetes.data))
         assert score < 1, (
```

Comment:
So here I would not find surprise that this `score=0`.

Reply:
I think that the threshold should be decreased indeed. Could you try:

```suggestion
        assert score == pytest.approx(0), (
```

to see if the test are passing. It might not be a good idea. I don't know if on 32 bits platform we could still have issue.